getnacelle/nacelle-js
# feat(examples/flash-sale): adds flash sale example

### WHY are these changes introduced?

Story [[LS-1560](https://nacelle.atlassian.net/browse/LS-1560)] 

### WHAT is this pull request doing?

Adds an example store demonstrating a flash sale, with flash content being provided by Contentful.

### How to Test

1. Run `npm run bootstrap` in the root.
2. In `examples/flash-sale`, add a `.env` file. Use the credentials provided below.
3. Run `npm run dev`.
4. Navigate to `/collections/clothing` to see the flash sale timer.

```
# Nacelle
NEXT_PUBLIC_NACELLE_STOREFRONT_ENDPOINT="https://storefront.api.nacelle.com/graphql/v1/spaces/f79296c6-b8dc-4b05-bc46-7305bc8728e3"
NEXT_PUBLIC_NACELLE_STOREFRONT_TOKEN="c5cd9e7c-a049-417b-a5ce-05d575632d2f"

# Shopify Checkout (for details, see: https://www.npmjs.com/package/@nacelle/shopify-checkout)
NEXT_PUBLIC_MYSHOPIFY_DOMAIN="prairie-wind-apparel-sales"
NEXT_PUBLIC_SHOPIFY_STOREFRONT_CHECKOUT_TOKEN="55db5e8a1325c82e275c5fc9301df711"
NEXT_PUBLIC_STOREFRONT_API_VERSION="2022-01"
```
